src/plugins/embeddable/public/tests/container.test.ts

```diff
@@ -174,25 +172,20 @@ test('Container.removeEmbeddable removes and cleans up', async (done) => {
 
   embeddable.updateInput({ lastName: 'Z' });
 
-  container
-    .getOutput$()
-    .pipe(skip(1))
-    .subscribe(() => {
-      const noFind = container.getChild<ContactCardEmbeddable>(embeddable.id);
-      expect(noFind).toBeUndefined();
+  container.removeEmbeddable(embeddable.id);
+  const promise = container.getOutput$().pipe(first()).toPromise();
```

Comment:
Can't we simplify these two lines to `await container.getOutput$().pipe(first()).toPromise()`?
I don't see anyone reusing the promise const.

Reply:
@lizozom you were right, Thank you :)